draft-scenarios/coops/the_ambushed_accord.tex

```diff
@@ -0,0 +1,109 @@
+\textit{The war had dragged on far too long. Armies were shattered, lands burned, and hope had thinned to a whisper. At last, the rival lords agreed to set aside their grievances and forge an Accord — a fragile peace born of desperation.\\Messengers rode beneath white banners, oaths were sworn, and for the first time in years, the clash of steel gave way to uneasy silence. Yet peace has its own enemies.\\
+As the allied hosts gathered in the valley of Laren’s Cross, dawn broke with the sound of horns — not of welcome, but of war.\\Out of the mist, unseen armies struck with deadly precision. Someone had betrayed the Accord.\\Now, surrounded and outnumbered, the fate of your alliance lies in your hands. Stand together… or fall alone.}  % no-check-caps
+\subsection*{\MakeUppercase{Scenario Length}}
+14 Rounds
+
+\subsection*{\MakeUppercase{Player Setup}}
+\textbf{Player Count:} 2-6
+
+\textbf{Starting Resources:} 2 \svg{gold}, 3 \svg{building_materials}, 1 \svg{valuables}
+
+\textbf{Starting Income:} 10 \svg{gold}, 0 \svg{building_materials}, 0 \svg{valuables}
+
+\textbf{Starting Units:}
+
+\begin{itemize}
+  \item 2 × Search (2) the \svg{bronze} Neutral Deck.
+\end{itemize}
+
+\textbf{Town Buildings:} City Hall.
+
+\textbf{Map Tile Pool:} Each player takes 2 random Far (II-III) Map Tiles and 2 random Near (IV-V) Map Tiles.
+
+\subsection*{\MakeUppercase{Map Setup}}
+Take the following Map Tiles ($P$ stands for the number of players) and arrange them as shown in the Scenario map layout:
+
+\begin{itemize}
+  \item $P$ × Starting (I) Map Tiles
+  \item 1 × Center (VI–VII) Map Tile
+\end{itemize}
+
+\subsection*{\MakeUppercase{Victory Conditions}}
+All players must defeat the Level VII Neutral Combat.
+
+\subsection*{\MakeUppercase{Timed Events}}
+\textbf{\nth{2} Round:}
+\begin{itemize}
+  \item Build either Camp (\svg{building_special_tent}) or Workshop (Gears) for free.
+\end{itemize}
+\textbf{\nth{4} Round:}
+\begin{itemize}
+  \item At the end of Round 4 \textbf{*One*} player must fight a Timed Event Fight (TEF) of 5x\svg{bronze}.
+\end{itemize}
+\textbf{\nth{8} Round:}
+\begin{itemize}
+  \item  At the end of Round 8 \textbf{*One*} player must fight a TEF of 4x\svg{silver}
+\end{itemize}
+\textbf{\nth{10} Round:}
+\begin{itemize}
+  \item  At the end of Round 10 \textbf{*One*} player must fight a TEF of 3x\svg{golden}.
+\end{itemize}
+\textbf{\nth{12} Round:}
+\begin{itemize}
+  \item  At the end of Round 12 \textbf{*One*} player must fight a TEF of 2x\svg{azure}.
+\end{itemize}
+\clearpage
+
+\subsection*{\MakeUppercase{Additional Rules}}
+\subsubsection*{\MakeUppercase{Difficulty Settings}}
+\textbf{Easy:}
+\begin{itemize}
+  \item Remove a Unit from each Timed Event Fight.
+\end{itemize}
+\textbf{Hard:}
+\begin{itemize}
+  \item Add a initiative stack token to a random Unit of the TEFs on Round 4, 8 and 10.
+\end{itemize}
+\textbf{Impossible:}
+\begin{itemize}
+  \item Add two random stack token to two Units of all TEF fights.
+\end{itemize}
```

Comment:
What about normal? We usually have tables for rulings like that.

Reply:
<img width="1164" height="1460" alt="image" src="https://github.com/user-attachments/assets/7ce4d065-c3af-49b9-9c52-c5b7fade3bbe" />